### PR TITLE
[DOCS-7123] Alfresco Content Accelerator 3.5 edits

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

**Comment by Amy-Hyland** [comment]
I believe the post ACA 3.5 changes have all been applied to the current branch.

**Review by katiebrill — APPROVED** [review]
Looks great to me. Thanks so much team!

**Review by anxumalo — CHANGES_REQUESTED** [review]
Looks good - see inline comments for suggested improvements

File: content-accelerator/latest/configure/actions.md
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply content-accelerator/latest/configure/actions.md:1]: Yes, this page does need a lot of edits.

File: content-accelerator/latest/configure/other-aca-admin-configs.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ Note also that there is currently no control over the picklist label/value or so
 
 ### DataDictionary Picklist
 
-**NOTE** - DataDictionary picklists are reserved for future use and are not currently supported.
+> **Note:** DataDictionary picklists are reserved for future use and are not currently supported.
```
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply content-accelerator/latest/configure/other-aca-admin-configs.md]: I changed this and I few other items. I restrained from editing more.

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**